### PR TITLE
Add messaging confluent scenario

### LINKED ISSUE #93: Migrate 301-quarkus-vertx-kafka from the Quarkus Beefy Test Suite
@pjgg I see the coverage is a bit similar to what we have in messaging/kafka-avro-reactive-messaging but without vertex.
Pls, analyse if the coverage here can be added in this module or create a new module under messaging/.

### PR DESCRIPTION
**Pull request by pjgg**
- Add Kafka confluent scenario
- Add docker-compose in order to run it manually
- Pending: Confluent Openshift scenario due to some PSI issues

Fix #93

### CODE REVIEW

File: messaging/kafka-avro-reactive-messaging/src/main/resources/application.properties [hunk 1]
```diff
@@ -1,19 +1,18 @@
...
-mp.messaging.incoming.channel-stock-price.connector=smallrye-kafka
-mp.messaging.incoming.channel-stock-price.topic=stock-price
-mp.messaging.incoming.channel-stock-price.specific.avro.reader=true
-mp.messaging.incoming.channel-stock-price.value.deserializer=io.apicurio.registry.utils.serde.AvroKafkaDeserializer
-mp.messaging.incoming.channel-stock-price.auto.offset.reset=earliest
-mp.messaging.incoming.channel-stock-price.enable.auto.commit=true
-mp.messaging.incoming.channel-stock-price.apicurio.registry.avro-datum-provider=io.apicurio.registry.utils.serde.avro.ReflectAvroDatumProvider
-quarkus.openshift.expose=true
+#Configuration file - Quarkus profile: Strimzi
+%dev.kafka.bootstrap.servers=localhost:9092
```
Comment (Sgitario) [inline messaging/kafka-avro-reactive-messaging/src/main/resources/application.properties:2]: Why did you move these properties into the `dev` profile?

File: messaging/kafka-avro-reactive-messaging/src/main/resources/docker-compose-strimzi.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
+version: '2'
```
Comment (Sgitario) [inline messaging/kafka-avro-reactive-messaging/src/main/resources/docker-compose-strimzi.yaml:1]: Is this file used?

**Review by Sgitario — CHANGES_REQUESTED** [review]

File: messaging/kafka-avro-reactive-messaging/src/main/resources/application.properties (hunk 1 above)
Comment (pjgg) [reply messaging/kafka-avro-reactive-messaging/src/main/resources/application.properties:2]: because those properties were used by our test and are not compatible with Confluent scenarios. On the other hand, I don't want to lose the ability to launch this app locally (manually).

File: messaging/kafka-avro-reactive-messaging/src/main/resources/docker-compose-strimzi.yaml (hunk 2 above)
Comment (pjgg) [reply messaging/kafka-avro-reactive-messaging/src/main/resources/docker-compose-strimzi.yaml:1]: if you launch this app manually then this file is very useful.

Comment (Sgitario) [reply messaging/kafka-avro-reactive-messaging/src/main/resources/docker-compose-strimzi.yaml:1]: Then, please add in the README.md information about how to use it and run the application.

File: pom.xml [hunk 3]
```diff
@@ -129,6 +130,11 @@
                 <artifactId>quarkiverse-apicurio-registry-client</artifactId>
                 <version>${quarkiverse.apicurio.registry.client.version}</version>
             </dependency>
+            <dependency>
+                <groupId>io.confluent</groupId>
+                <artifactId>kafka-avro-serializer</artifactId>
+                <version>${confluent.kafka-avro-serializer.version}</version>
+            </dependency>
```
Comment (Sgitario) [inline pom.xml:137]: The dependency is here, but the repository where these dependencies are in is declared in a submodule. This is wrong as it would require other submodules to register the repository every time a submodule needs a confluent dependency.

**Review by Sgitario — CHANGES_REQUESTED** [review]

**Comment by Sgitario** [comment]
The PR title contains [WIP], are you planning to add more things to this?

File: pom.xml [hunk 4]
```diff
@@ -25,6 +25,7 @@
         <apache-httpclient-fluent.version>4.5.13</apache-httpclient-fluent.version>
         <apicurio.registry.utils.serde.version>1.3.2.Final</apicurio.registry.utils.serde.version>
         <quarkiverse.apicurio.registry.client.version>0.0.2</quarkiverse.apicurio.registry.client.version>
+        <confluent.kafka-avro-serializer.version>6.0.0</confluent.kafka-avro-serializer.version>
```
Comment (Sgitario) [inline pom.xml:28]: The latest version is `6.2.0`, please update it here otherwise Dependabot will create a separated PR for doing only this.

**Review by Sgitario — CHANGES_REQUESTED** [review]

**Review by Sgitario — APPROVED** [review]
